cmd/mailbox-controller/main.go

```diff
@@ -70,15 +70,13 @@ func main() {
 	fs.IntVar(&concurrency, "concurrency", concurrency, "number of syncs to run in parallel")
 	fs.StringVar(&espwPath, "espw-path", espwPath, "the pathname of the edge service provider workspace")
 
-	inventoryClientOpts := clientopts.NewClientOpts("inventory", "access to APIExport view of SyncTarget objects")
-	inventoryClientOpts.SetDefaultCurrentContext("root")
+	rootClientOpts := clientopts.NewClientOpts("root", "access to the root workspace")
```

Comment:
Don't forget the updates to https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/mailbox-controller/#usage